frontend/src/components/workspace/StartApplicationModal.tsx

```diff
@@ -25,15 +26,19 @@ const StartApplicationModal = ({
   const { user } = useUser();
   const router = useRouter();
   const t = useTranslations("OpportunityListing");
+  const headerTranslation = useTranslations("HeaderLoginModal");
   const modalId = "start-application";
   const [validationError, setValidationError] = useState<string>();
   const [savedSearchName, setSavedSearchName] = useState<string>();
   const [error, setError] = useState<string>();
   const [loading, setLoading] = useState<boolean>();
+  const token = user && user.token ? user.token : null;
```

Comment:
You could simplify this a bit with:
```suggestion
  const token = user?.token || null;
```

Reply:
Great catch, thanks.